src/docs/src/best-practices/reverse-proxies.rst

```diff
@@ -221,6 +221,22 @@ as ``http(s)://domain.com/couchdb/db1/doc1`` are proxied to
 
     }
 
+To proxy the URL ``http(s)://domain.com/couchdb`` to
+``http://localhost:5984`` so that requests appended to the subdirectory, such
+as ``http(s)://domain.com/couchdb/db1/doc1`` are proxied to
+``http://localhost:5984/db1/doc1``, you have to configure a route in Caddy and strip the subdirectory from the target URI with ``uri strip_prefix``:
```

Comment:
the linter complains about this long line (). please add a line break (max. 90 chars).